VedaDom/Auto-Mart
# #166757276 Get all available car [feature]

#### What does this PR do?
This PR adds contains an API for getting all car available if the user is not admin gets cars with sold and unsold status.
#### Description of Task to be completed?
- `GET` api/v1/car
- A token must be provided to check if a user is `Admin`
#### How should this be manually tested?
- clone the repo and switch to the branch `ft-get-all-available-car-166757276`
- install the dependencies use the commands, `npm install` and  `npm start` to start the server
- register an admin and use the endpoint `GET /api/v1/car/:id` on postman to test
- you should get a successful response:
    {  
                 status:  200,
                 data:  [...]
    }
#### What are the relevant pivotal tracker stories?
[Get all available cars #166757276](https://www.pivotaltracker.com/story/show/166757276)
